server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailQueryMethodContract.scala

```diff
@@ -2997,7 +2997,7 @@ trait EmailQueryMethodContract {
 
     val message1: Message = Message.Builder
       .of
-      .setTo("aaa@domain.tld")
+      .setTo("\"aaa\" <aaa@domain.tld>")
```

Comment:
Do you think we get a bug with parsing MailAddress in Postgres module?

I tend to think fix Postgres than change the contract here

Reply:
the error still occurred when I debug this test with cassandra implementation. Do not know why this test has never failed in master build?

Reply:
>  Do not know why this test has never failed in master build?

CI only runs only `BasicFeature` (to save time)
https://github.com/apache/james-project/pull/2116